public/jira/theme/fetcher.ts

```diff
@@ -0,0 +1,103 @@
+import { getTextColorUsingAPCA } from "../../utils/color";
+import { AppStorage } from "../storage/common";
+
+export const defaultTheme = {
+  complete: {
+    label: "Complete",
+    description: `End date in the past`,
+    backgroundColor: "#22A06B",
+    backgroundCssVar: "--complete-color",
+    textColor: "#FFFFFF",
+    textCssVar: "--complete-text-color",
+  },
+  blocked: {
+    label: "Blocked",
+    description: `Has Jira status of "blocked" or label of "blocked"`,
+    backgroundColor: "#E2483D",
+    backgroundCssVar: "--blocked-color",
+    textColor: "#FFFFFF",
```

Comment:
Should `textColor` be derived from `backgroundColor`? I suppose we could eventually allow people to configure it. But I'm surprised a `textColor` value is needed in this construct and it wouldn't be derived later from `backgroundColor`.

Reply:
Thinking about it a bit more, i like just deriving it at the var level, I updated the code